packages/storage/src/components/ImageGallery/styled.js

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components';
+import { FileSelector } from 'react-rainbow-components';
+
+export const StyledContainer = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    padding: 10px 20px;
+`;
+
+export const StyledFileContainer = styled.div`
+    height: 190px;
+    width: 200px;
+    margin-right: 10px;
+    margin-bottom: 10px;
+    overflow: hidden;
+`;
+
+export const StyledFileSeletor = styled(FileSelector)`
+    display: block;
+    width: 200px;
+    height: 160px;
+    margin-right: 10px;
+    margin-bottom: 10px;
+
+    > div {
```

Comment:
we should not make introspection on the components, we have to follow the best practices